PatientRegistrationApp/src/components/EligibilityQuestions.vue

```diff
@@ -0,0 +1,201 @@
+          <v-col cols="12" sm="12" md="6" lg="6"><v-radio label="Other" value="workPosition31"></v-radio></v-col>
+        </v-radio-group>
+      </v-col>
+    </v-row>
+    <v-row no-gutters>
+      <v-divider></v-divider>
+    </v-row>
+    <!-- Eligibility Question 6 -->
+    <v-row no-gutters>
+          <div><span class="red--text"><strong>* </strong></span>6. Do you have any of the following high-risk conditions? (Check all that apply)</div>
+    </v-row>
+    <v-row no-gutters>
+        <!-- <v-radio-group v-model="eligibilityQ6" row required> -->
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Cancer" value="condition1"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Chronic kidney disease" value="condition2"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="COPD (chronic obstructive pulmonary disease)" value="condition3"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Down Syndrome" value="condition4"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Heart conditions, such as heart failure, coronary artery disease, or cardiomyopathies" value="condition5"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Immunocompromised state (weakened immune system) from solid organ transplant or from blood or bone marrow transplant, immune deficiencies, HIV, use of corticosteroids, or use of other immune weakening medicines" value="condition6"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Obesity (body mass index [BMI] of 30 kg/m2 or higher but < 40 kg/m2)" value="condition7"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Severe Obesity (BMI ≥ 40 kg/m2)" value="condition8"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Pregnancy" value="condition9"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Sickle cell disease" value="condition10"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Smoking" value="condition11"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="Type 2 diabetes mellitus" value="condition12"></v-checkbox></v-col>
+          <v-col cols="12" sm="12" md="6" lg="6"><v-checkbox v-model="eligibilityQ6" label="None of the above" value="condition13"></v-checkbox></v-col>
```

Comment:
Is it possible/easy enough to unselect and disable the checkboxes when None of the above is selected?